tests/db_functions/text/test_reverse.py

```diff
@@ -0,0 +1,45 @@
+from django.db import connection
+from django.db.models import CharField
+from django.db.models.functions import Length, Reverse, Trim
+from django.test import TestCase
+from django.test.utils import register_lookup
+
+from ..models import Author
+
+
+class ReverseTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.john = Author.objects.create(name='John Smith', alias='smithj')
+        cls.elena = Author.objects.create(name='Élena Jordan', alias='elena')
+        cls.python = Author.objects.create(name='パイソン')
+
+    def test_null(self):
+        author = Author.objects.annotate(backward=Reverse('alias')).get(pk=self.python.pk)
+        self.assertEqual(author.backward, '' if connection.features.interprets_empty_strings_as_nulls else None)
+
+    def test_basic(self):
+        authors = Author.objects.annotate(backward=Reverse('name')).order_by('name')
+        self.assertQuerysetEqual(
+            authors,
+            [
+                ('John Smith', 'htimS nhoJ'),
```

Comment:
On MySQL and PostgreSQL, Élena and John sort in the opposite order. I think we can leave off `order_by()` and ignore ordering?

Reply:
Agreed :+1: